app/src/web.ts

```diff
@@ -0,0 +1,40 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type { AppInfo, AppPlugin, AppLaunchUrl, AppState } from './definitions';
+
+export class AppWeb extends WebPlugin implements AppPlugin {
+  constructor() {
+    super({ name: 'App' });
+    if (typeof document !== 'undefined') {
+      document.addEventListener(
+        'visibilitychange',
+        this.handleVisibilityChange.bind(this),
+        false,
+      );
+    }
+  }
+
+  exitApp(): never {
+    throw new Error('Method not implemented.');
```

Comment:
Would it be a bad idea to call `window.close()` ?

Reply:
I've tried and it doesn't close the window, just shows `Scripts may close only the windows that were opened by them.`